extensions/mssql/src/hdfs/webhdfs.ts

```diff
@@ -780,10 +780,6 @@ export class WebHDFS {
 				stream.pipe(upload);
 				stream.resume();
 			}
-			if (error && !response) {
-				// request failed, and req is not accessible in this case.
-				throw this.parseError(undefined, undefined, error);
-			}
```

Comment:
Here's the fix for the error handling. Without it we were getting an unhandled promise rejection and the task would never update since the stream was never completing. @kevcunnane see any issues with this?

Reply:
It seems like you tested this, so seems fine to me. Thanks for catching.